code/src/components/BackButton.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+
+import { useHistory } from "react-router-dom";
+import arrowLeft from '../assets/ic_24_arrowLeft.svg'
+
+export const BackButton = () => {
+    let history = useHistory();
+    return (
+        <>
```

Comment:
No need for React Fragment if you return only one element (in your case, `<button />`)